models/Asset/Image/Thumbnail/Config.php

```diff
@@ -20,8 +20,6 @@
 use Pimcore\Tool\Serialize;
 
 /**
- * @method \Pimcore\Model\Asset\Image\Thumbnail\Config\Dao getDao()
```

Comment:
Why removed that? Still valid right?

Reply:
because Pimcore\Model\Asset\Image\Thumbnail\Config now has its own implementation of save. So the annotation would be invalid.